server/conf/application.dev.conf

```diff
@@ -63,11 +63,14 @@ esri_address_service_area_validation_labels = ["Seattle"]
 esri_address_service_area_validation_ids = ["Seattle"]
 esri_address_service_area_validation_attributes = ["CITYNAME"]
 
+# Allow any IP for local development
+api_keys_ban_global_subnet = false
+
 # Allow dev sessions to last 5 days
 maximum_session_duration_minutes = 7200
 
 api_generated_docs_enabled = true
 program_filtering_enabled = true
 name_suffix_dropdown_enabled = true
 session_timeout_enabled = false
-custom_theme_colors_enabled = true
```

Comment:
No matter what I do I cannot get this newline difference to go away. Not locally in my editor. Not on github's online editor. I'm done trying.